backend/fms_core/models/full_sample.py

```diff
@@ -80,6 +86,14 @@ class FullSample(models.Model):
     updated_by = models.ForeignKey(User, null=False, blank=True, related_name="%(app_label)s_%(class)s_modification", on_delete=models.DO_NOTHING)
     deleted = models.BooleanField(default=False, help_text="Whether this instance has been deleted.")
 
+    @property
+    def extracted_from(self) -> "Sample":
```

Comment:
Maybe just call sample.extracted_from() ? If sample.extracted_from is not working we need to fix it ...